packages/teams-js/src/public/appNotification.ts

```diff
@@ -0,0 +1,98 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the appNotifications specific part of the SDK
+ */
+export namespace appNotification {
+  /**
+   * Data structure to represent appNotification information
+   */
+  export interface NotificationDisplayParam {
+    /**The heading of the notiification to be delivered,it would have a maximum string length of 20*/
```

Comment:
We usually use this comment pattern, instead of one liner
```
/**
 * some comment
 */
  ```

Reply:
JFYI it's stronger than "usually." It's required that all our public interfaces/properties/functions be documented using the typedoc syntax so it can be published to the Microsoft docs site. I suspect that if you don't do this, the build will fail (it should fail) -- it didn't fail in the pipeline for this run because it failed and aborted when it couldn't find the beachball changefile.

FYI you can and should run the build locally (using `pnpm build`) to help validate this before publishing the PR. I don't expect that building locally will catch the beachball problem, but it should catch the incorrect documentation problem.